tests/unit/test_particle.py

```diff
@@ -70,7 +79,7 @@ def test_exceptions(self):
                 parity=-1,
                 c_parity=-1,
                 g_parity=-1,
-                isospin=(0, 0),
+                isospin=(Fraction(0), Fraction(0)),
```

Comment:
See remark about `SupportsFloat`. Then you can write

```suggestion
                isospin=(0, 0),
```